core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -160,12 +164,14 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
   /**
    * Operator alias for `orElse`.
    */
+  // TODO: This usually the monoid operator
   final def <>[R1 <: R, E2, A1 >: A](that: => ZIO[R1, E2, A1])(implicit ev: CanFail[E]): ZIO[R1, E2, A1] =
     orElse(that)
 
   /**
    * A symbolic alias for `raceEither`.
    */
+  @deprecated("use raceEither", "2.0.0")
   final def <|>[R1 <: R, E1 >: E, B](that: ZIO[R1, E1, B]): ZIO[R1, E1, Either[A, B]] =
```

Comment:
I would be inclined to keep this since `race` is actually a relatively commonly used operator on `ZIO` and it has a nice symmetry with `<&>`.